docs/redis_unavailability.md

```diff
@@ -0,0 +1,4 @@
+## Redis Unavailability
+
+If redis server fails, it will `console.error` this fact, and it will keep retrying to connect every 30 seconds. Once connection is resumed
```

Comment:
why are these separate readmes ?

Reply:
I think it would be better if we do something like http://grapher.cultofcoders.com

Reply:
Good point, but people will care about failover, so it can send also a positive signal

Reply:
What if we write "Designed to handle failover" as one of the features, and provide a link to more information?

BTW - can you get Pub/Sub to regress to polling the database whenever Redis becomes unavailable?

Reply:
@msavin indeed sounds better.

To do that, it would require some big design changes.